### ISSUE TITLE
Hi

### ISSUE THREAD

**Issue by tbjgolden** [newshipt/shiptspell#7]

**Comment by dneilroth** [comment]
hey @tbjgolden

**Comment by dneilroth** [comment]
`Professional mammal` 🐒

**Comment by tbjgolden** [comment]
Hey @dneilroth how's everything going?

**Comment by tbjgolden** [comment]
How's Checkr?

**Comment by hatched-danny** [comment]
Hello friends.

**Comment by dneilroth** [comment]
it's good, I spend most of my day perusing the Shipt github repos I still have access to. How is the thing that you are doing now? lets a grab a beer sometime!

**Comment by tbjgolden** [comment]
Yeah good, working with Devin and Danny in the Mission - come along to our office some time 🍻 
If you ever need to get hold of me at all this might actually be the easiest way since I've killed my facebook 😂
